### DISCUSSION TITLE
Request/Feature: VT100 Font for Terminal

### DISCUSSION THREAD

**Discussion by guidol70** [fdivitto/FabGL#175] [Ideas]
Is there a chance to get a VT100 font like on the following
VT100 CRT Emulation?:
https://github.com/larsbrinkhoff/terminal-simulator

The emulations does run the VT100 Firmware in a 8080-Emulator.
So it sounds like a Z80-Emulator like on the VGA32 should also be a possible VT100?
Maybe only with a special font or like the complete emulation like the other github-project?

![Before_scrolling](https://user-images.githubusercontent.com/5121124/122541968-e1ca3600-d032-11eb-9858-015df065f15d.png)

**Comment by guidol70** [comment]
this one here could be more easy to convert. A [vt52 font in .otf](https://github.com/fritzm/vt52/blob/master/fonts/vt52.otf) format.
(I also converted it online to .ttf)
![vt52_ttf_RunCPM](https://user-images.githubusercontent.com/5121124/125645253-038f346d-b67b-420d-83e4-997686f964e3.jpg)
[vt52_ttf.zip](https://github.com/fdivitto/FabGL/files/6817030/vt52_ttf.zip)

**Comment by fdivitto** [comment]
do you like to have a true VT100 font (meaning the character set) or just the CRT effect?
FabGL terminal needs a fixed width font, bitmapped. The common used format for this kind of font is "FON".

Using a real VT100 font means you will have a 8x9 font, that using 640x480 resolution results to 80x53 screen size.

**Comment by guidol70** [comment]
Thats a good question ;) - because I like the VT100 Font, but I dont know if its that nice without the CRT-Effect.

If I have do decide in the first place, I would prefer CRT-Mode as the first wish.
If I take a look at the screenshots above the VT100 and the VT52 Font does use CRT-lines

I did try to decode the hexcodes in the font-.h files but didnt get a real char if I decode hex to binary and set the pixel like in the bit-pattern 
(did try did try 8x8 font - Line 65 of the 256 Lines to get a A).

So the 8x8 A (without Scanlines) from the VT52 Font
```
........
....#...
...#.#..
..#...#.
.#.....#
.#######
.#.....#
.#.....#

```
could be a 8x14 "scanline" A?

```
........
....#...
........
...#.#..
........
..#...#.
........
.#.....#
........
.#######
........
.#.....#
........
.#.....#
```
